documentation/source/release_notes.rst

```diff
@@ -4,6 +4,36 @@ Release Notes
 
 All releases are available from the `GitHub Releases Page <https://github.com/cocotb/cocotb/releases>`_.
 
+cocotb 1.3
+==========
+
+Released on 1 January 2020
+
+New features
+------------
+- :func:`cocotb.triggers.with_timeout`, for a shorthand for waiting for a trigger with a timeout. (:pr:`1119`)
+- New ``timeout_time`` and ``timeout_unit`` arguments to :func:`cocotb.test`, for adding test timeouts. (:pr:`1119`)
+- The ``expect_error`` argument to :func:`cocotb.test` now accepts a specific exception type. (:pr:`1116`)
+- :envvar:`COCOTB_RESULTS_FILE`, to allow configration of the xunit XML output filename.  (:pr:`1053`)
+- :class:`cocotb.triggers.Timer` can now be used with :class:`fractions.Fraction` instances, allowing constructs like ``Timer(Fraction(1, 10000), units='sec')`` as an alternate spelling for ``Timer(100, units='us')``. (:pr:`1114`)
+- Support for the :ref:`Verilator` simulator (version 4.020 and above).
+- A new ``bus_separator`` argument to :class:`cocotb.drivers.BusDriver`. (:pr:`1160`)
+- A new ``start_high`` argument to :meth:`cocotb.clock.Clock.start`. (:pr:`1036`)
+
+Notable changes and bug fixes
+-----------------------------
+- ``DeprecationWarning``\ s are now shown in the output by default.
+- Tracebacks are now preserved correctly for exceptions in Python 2. The tracebacks in all python versions are now a little shorter.
+- :func:`cocotb.external` and :func:`cocotb.function` now work more reliably and with fewer race conditions.
+- A failing ``assert`` will be considered a test failure. Previously, it was considered a test _error_.
+- :meth:`~cocotb.handles.NonConstantObject.drivers` and :meth:`~cocotb.handles.NonConstantObject.drivers` now work correctly in python 3.7 onwards.
```

Comment:
This is pointing two times to the same method (and doesn't hyperlink).

Reply:
`loads`, whoops